Tasks/google-play-promote/package.json

```diff
@@ -11,7 +11,6 @@
   "dependencies": {
     "bluebird": "^2.10.1",
     "googleapis": "^2.1.5",
-    "minimatch": "^3.0.0",
```

Comment:
I checked vso-task-lib and it specifies its own dependency for minimatch. Checked with Peter too.